api/soap/mc_config_api.php

```diff
@@ -33,19 +33,19 @@
 function mc_config_get_string( $p_username, $p_password, $p_config_var ) {
 	$t_user_id = mci_check_login( $p_username, $p_password );
 	if( $t_user_id === false ) {
-		return mci_soap_fault_login_failed();
+		return mci_fault_login_failed();
 	}
 
 	if( !mci_has_readonly_access( $t_user_id ) ) {
-		return mci_soap_fault_access_denied( $t_user_id );
+		return mci_fault_access_denied( $t_user_id );
 	}
 
 	if( config_is_private( $p_config_var ) ) {
-		return SoapObjectsFactory::newSoapFault( 'Client', 'Access to \'' . $p_config_var . '\' is denied' );
+		return ApiObjectFactory::fault( 'Client', 'Access to \'' . $p_config_var . '\' is denied', 403 );
```

Comment:
Good point.  I have done the following:
- Added constants for http status codes in core/constants_inc.php
- Create factory methods for each status code.  Hence raising an error now looks as follows:

```
return ApiObjectFactory::faultForbidden( 'some reason why' );
```